auction-server/migrations/20250726182247_auction_creation_time_idx.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX auction_creation_time_idx ON auction (creation_time);
```

Comment:
this is going to be a big migration, but it may be necessary to efficiently delete rows from auction long-term. we could optionally make this query `CREATE INDEX CONCURRENTLY` which allows the index to [be created non-atomically](https://www.postgresql.org/docs/current/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY) and without blocking writes